frontend/src/features/properties/filter/PropertyFilterAgencyOptions.tsx

```diff
@@ -0,0 +1,63 @@
+import { useFormikContext } from 'formik';
+import React, { useEffect } from 'react';
+import { Select, SelectOption } from '../../../components/common/form';
+import { IPropertyFilter } from './IPropertyFilter';
+import { ParentSelect } from 'components/common/form/ParentSelect';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+
+interface IPropertyFilterAgencyOptions {
+  disabled?: boolean;
+  agencies: SelectOption[];
+}
+
+/**
+ * Provides a dropdown that populates includeAllProperties and controls the agencies input.
+ */
+export const PropertyFilterAgencyOptions: React.FC<IPropertyFilterAgencyOptions> = ({
+  disabled,
+  agencies,
+}) => {
+  const state: { options: any[] } = {
+    options: [
+      { label: 'My Agencies', value: false },
+      { label: 'All Government', value: true },
+    ],
+  };
+  const {
+    setFieldValue,
+    values: { includeAllProperties },
+  } = useFormikContext<IPropertyFilter>();
+  const keycloak = useKeycloakWrapper();
+
+  useEffect(() => {
+    if (includeAllProperties === false) {
+      setFieldValue('agencies', keycloak.agencyId);
```

Comment:
this sets the default agency when the select is changed from All Government to My Agencies.